### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ReaLoretta**
geageahehaefge

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # complete-trash
-this is rubbish
+this is love
```
Comment (ReaLoretta) [inline README.md:2]: love is rubbish

Comment (ReaLoretta) [reply README.md:2]: @Restuta

**Review by Restuta — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (ReaLoretta) [reply README.md:2]: Well sure

Comment (ReaLoretta) [reply README.md:2]: @Restuta do you like this?

Comment (ReaLoretta) [reply README.md:2]: @Restuta hi

Comment (ReaLoretta) [reply README.md:2]: @Restuta what about *this*?

Comment (ReaLoretta) [reply README.md:2]: @Restuta check out this code:
```js
const x = () => 'hi;
```

Comment (Restuta) [reply README.md:2]: ok

Comment (ReaLoretta) [reply README.md:2]: sure?

Comment (Restuta) [reply README.md:2]: @Restuta hi

Comment (Restuta) [reply README.md:2]: @Restuta hello

Comment (Restuta) [reply README.md:2]: hi

**Comment by Restuta** [comment]
@Restuta hello

**Comment by Restuta** [comment]
sdf

**Review by Restuta — APPROVED** [review]
sdfsdf

**Pull request closed without merging**